packages/components/src/components/form/SelectBar/SelectBar.tsx

```diff
@@ -170,7 +170,7 @@ export type SelectBarProps<V extends ValueTypes> = {
 } & AllowedFrameProps;
 
 // Generic type V is determined by selectedOption/options values
-export const SelectBar: <V extends ValueTypes>(props: SelectBarProps<V>) => JSX.Element = ({
```

Comment:
:thought_balloon:  no need to specify return type `JSX.Element`. React throws TS error when calling a non-compliant function as `<Component>`.
Though personally I'd like declarative typing for components - if the function returns invalid type, TS should ideally throw that in the component definition, not in the JSX that's calling it. But:
1. we don't do that almost anywhere :cry:
2. `JSX.Element` is deprecated anyway, `ReactNode` would be better, and `React.FC` best...
3. in this case it'd break inference of `V` if we had `const SelectBar: FC<whatever> = function`